.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+
+node_js: "6"
+
+addons:
+  apt:
+    packages:
+      - xvfb
+      - fakeroot
+      - dpkg
+      - libdbus-1-dev
+      - libglib2.0-dev
+      - dbus
+      
+before_install:
+  - cd debian/
+  
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  
```

Comment:
The app repo has GUI test so it is covered there.  Maybe we should delete the before_script if you think. the `sleep 3 ` looks like a recepie for flakyness (failed builds due to magic numbers).